spring-cloud-gcp-dependencies/pom.xml

```diff
@@ -44,6 +45,17 @@
 				<version>${project.version}</version>
 			</dependency>
 			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-gcp-stream-core</artifactId>
```

Comment:
In other binders, for core modules, there is a standard naming convention we follow. for ex: `spring-cloud-stream-binder-core`. You may want to consider something similar for consistency reasons as this is a binder implementation.

Reply:
Renamed the two sub-modules to `spring-cloud-stream-binder-gcp-pubsub` and `spring-cloud-stream-binder-gcp-pussub-core` to keep up naming consistency.